tests/Benchmarks/System.Buffers/ToSpan.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Buffers;
+using System.Buffers.Tests;
+using System.Collections.Generic;
+using System.Collections.Sequences;
+using System.Linq;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+
+namespace Benchmarks.System.Buffers
+{
+    public class ToSpan
+    {
+        public static ReadOnlySequence<byte> Sequence = BufferFactory.Create(Enumerable.Repeat(0, 10000000)
+            .Select(x => new Memory<byte>(new byte[] { 0 })));
+
+        [Benchmark]
+        public void NewToSpan()
+        {
+            var Result = Sequence.ToSpan();
```

Comment:
nit: instead of assigning it to a local, return it, if possible.

Also, don't use var since it isn't super clear from the right hand side of the expression, what the type is.
```suggestion
            ReadOnlySpan<byte> result = Sequence.ToSpan();
```
OR
```suggestion
            return Sequence.ToSpan();
```